src/renderers/dom/client/utils/DOMChildrenOperations.js

```diff
@@ -247,6 +248,39 @@ var DOMChildrenOperations = {
     }
   },
 
+  startObservation: function() { 
+    var config = { attributes: false, childList: true, characterData: false, subtree: true };
+    var MutationObserver = window.MutationObserver || function() {};
+    MutationObserver.prototype.observe = MutationObserver.prototype.observe || function() {};      
+    MutationObserver.prototype.disconnect = MutationObserver.prototype.disconnect || function() {};      
```

Comment:
I want to make a devtool hook as suggested but I've never made a react devtool hook before. Could you elaborate on the general process for making a devtool?